### PR TITLE
Feature/ab#49301 update upload file to use correct text

### PR DESCRIPTION
**Pull request by tc-chris**

### CODE REVIEW

File: CDTS Blazor/Pages/UploadDocument.razor [hunk 1]
```diff
@@ -175,14 +199,18 @@
...
+
             model.UploadedFiles.Add(new UploadedFile { SelectedFile = item, Description = model.FileDescription });
 
             model.FileDescription = string.Empty;
         }
 
         StateHasChanged();
 
-        await Task.Delay(100);
+        await Task.Delay(0);
```
Comment (AlexandreSanscartier) [inline CDTS Blazor/Pages/UploadDocument.razor:213]:
```suggestion

```

File: CDTS Blazor/Resources/Pages/UploadDocument.fr.resx [hunk 2]
```diff
@@ -197,4 +197,10 @@
   <data name="Next" xml:space="preserve">
     <value>Procéder à la révision</value>
   </data>
+  <data name="AddDocument" xml:space="preserve">
+    <value>Ajouter un document</value>
+  </data>
+  <data name="AddNewDocument" xml:space="preserve">
+    <value>Ajouter un document</value>
```
Comment (AlexandreSanscartier) [inline CDTS Blazor/Resources/Pages/UploadDocument.fr.resx:204]: I'm assuming this has been verified by UX, but could you confirm @tc-chris?

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
A few comments to check before merging back in.

File: CDTS Blazor/Pages/UploadDocument.razor (hunk 1 above)
Comment (tc-chris) [reply CDTS Blazor/Pages/UploadDocument.razor:213]: We can remove it

**Review by AlexandreSanscartier — APPROVED** [review]

**Pull request merged**